template.html

```diff
@@ -378,19 +384,42 @@ <h3>{{topic.others.by-us-title}}</h3>
     </footer>
 
     <script>
-      // ⚠ Contents of this script block are hashed and used in
-      // the CSP meta tag. Any change (including white space) will
-      // require update of the CSP tag. All this to prevent the
-      // need for 'unsafe-inline'.
+      // ⚠ The contents of this script block are hashed and used in the CSP meta tag. 
+      // Any change (including white space) will require an update of the CSP tag. 
+      // All this to prevent the need for 'unsafe-inline'.
+
       document.addEventListener("DOMContentLoaded", () => {
-        [...document.querySelectorAll(".details-toggle")].forEach(element => {
-          const anchor = element.dataset.toggleTarget;
-          element.addEventListener('click', () => {
-            document.getElementById('viewToggle').classList.toggle('view-overview');
-            location.hash = anchor;
-          });
-        });
+
+        const url = new URL(window.location.href);
+        const button = document.querySelector('[data-toggle="summary"]');
+
+        if (url.searchParams.get('summary') === 'true') {
+          document.getElementById('viewToggle').classList.toggle('view-summary');
+          button.setAttribute('aria-checked', 'false');
+          button.setAttribute('aria-expanded', 'false');
```

Comment:
Is there a reason we need to set both of these attributes? (Also in the `toggleDetailView`) It feels unnecessary, but could be that it's good for a11y of course.

Reply:
Nice catch. I've looked it up and it seems like only using the `aria-expanded` attribute is the way to go. While technically it isn't incorrect to use `aria-checked` here (since it does toggle something), using both attributes at the same time might cause confusion in screenreaders and since `aria-expanded`'s purpose is more targeted towards toggling content, I've opted for that one.